projects/common/src/index.ts

```diff
@@ -162,12 +162,23 @@ export interface AtomElement {
     css?: string[]
     js?: string[]
 }
+
+export interface MediaAtomElement {
```

Comment:
Where is the id of the atom? This needs to be passed through for us to render the video.

`https://embed.theguardian.com/embed/atom/media/${atomId}`

Reply:
Sorry, added